src/js/Complete.js

```diff
@@ -0,0 +1,35 @@
+import Ls from './Ls.js';
+import * as globals from './globals.js';
+import RemoveItems from './Remove.js';
+
+class Complete {
+  Ls = new Ls();
+
+  RemoveItems = new RemoveItems();
+
+  clearCompleted = () => {
```

Comment:
- [ ] Good work implementing this function, however, you should use ``` Array.filter()``` method to implement the function for the "Clear all completed" button. Kindly make these changes to your code so it meets the requirements